tests/chart_tests/test_airflow_operator.py

```diff
@@ -114,3 +114,60 @@ def test_airflow_operator_webhooks(self, kube_version):
         assert "ValidatingWebhookConfiguration" == docs[1]["kind"]
         assert "release-name-airflow-operator-mutating-webhook-configuration" == docs[0]["metadata"]["name"]
         assert "release-name-airflow-operator-validating-webhook-configuration" == docs[1]["metadata"]["name"]
+
+    def test_airflow_operator_runtimeversion(self, kube_version):
+        """""Test Airflow Operator runtimeversion""" ""
+        random_json = {
+            "runtimeVersions": {
+                "4.2.5": {"metadata": {"airflowVersion": "2.4.2", "channel": "stable", "releaseDate": "2023-01-15"}},
+                "5.0.0": {"metadata": {"airflowVersion": "2.5.0", "channel": "stable", "releaseDate": "2023-03-20"}},
+            }
+        }
+        docs = render_chart(
+            validate_objects=False,
+            kube_version=kube_version,
+            values={
+                "global": {
+                    "airflowOperator": {"enabled": True},
+                },
+                "airflow-operator": {"airgapped": True, "runtimeVersions": {"versionsJson": random_json}},
+            },
+            show_only=["charts/airflow-operator/templates/configmap/runtime-versions.yaml"],
+        )
+        assert len(docs) == 1
+        template = docs[0]
+        assert template["apiVersion"] == "v1"
+        assert template["data"]["versions.json"] == random_json
+        assert template["kind"] == "ConfigMap"
+        assert template["metadata"]["name"] == "release-name-runtime-version-config"
+        assert template["metadata"]["labels"]["tier"] == "operator"
```

Comment:
can you use the same format used in other test cases to have consistency 
```suggestion
        doc = docs[0]
        assert doc["apiVersion"] == "v1"
        assert doc["data"]["versions.json"] == random_json
        assert doc["kind"] == "ConfigMap"
        assert doc["metadata"]["name"] == "release-name-runtime-version-config"
        assert doc["metadata"]["labels"]["tier"] == "operator"
```